student/utils.py

```diff
@@ -50,6 +50,14 @@ def get_student(request):
         return None
 
 
+def get_yeet(request):
+    """
+    Returns:
+        (:obj:`Yeet`): My Yeet object
+    """
+    return Yeet.objects.get(id=1)
+
```

Comment:
LGTM! 

Only comment I have is identical to what I commented on Sebastian's PR [#123] - So check that out if you want to see an example of how to use relations in Django to tie models together :)